index.html

```diff
@@ -2385,24 +2392,281 @@ <h2>
+        a89d05e72f                             #       "\xa8\x9d\x05\xe7/"
+  69                                           #   text(9)
+     70726f746563746564                        #     "protected"
+  a0                                           #   map(0)
+  6a                                           #   text(10)
+     7369676e617475726573                      #     "signatures"
+  81                                           #   array(1)
+     a3                                        #     map(3)
+        69                                     #       text(9)
+           70726f746563746564                  #         "protected"
+        66                                     #       text(6)
+           613130313236                        #         "a10126"
+        69                                     #       text(9)
+           7369676e6174757265                  #         "signature"
+        78 80                                  #       text(128)
+           65326165616664343064363964313964    #         "e2aeafd40d69d19d"
+           66653665353230373763356437666634    #         "fe6e52077c5d7ff4"
+           65343038323832636265666235643036    #         "e408282cbefb5d06"
+           63626634313461663265313964393832    #         "cbf414af2e19d982"
+           61633435616339386238353434633930    #         "ac45ac98b8544c90"
+           38623435303764653165393062373137    #         "8b4507de1e90b717"
+           63336433343831366665393236613262    #         "c3d34816fe926a2b"
+           39386635336166643266613066333061    #         "98f53afd2fa0f30a"
+        6b                                     #       text(11)
+           756e70726f746563746564              #         "unprotected"
+        a1                                     #       map(1)
+           63                                  #         text(3)
+              6b6964                           #           "kid"
+           78 85                               #         text(133)
+              6469643a697069643a313244334b6f6f #           "did:ipid:12D3Koo"
+              574d4864727a6377706a6264725a7335 #           "WMHdrzcwpjbdrZs5"
+              47477145524176636771583362356470 #           "GGqERAvcgqX3b5dp"
+              7550745061396f7436397965773b6970 #           "uPtPa9ot69yew;ip"
+              69643a6b65792d69643d626166797265 #           "id:key-id=bafyre"
+              6963756274783577716f336e6f736334 #           "icubtx5wqo3nosc4"
+              63617a726b6374666877643672657765 #           "cazrkctfhwd6rewe"
+              7a6770776f6534737769726c73346562 #           "zgpwoe4swirls4eb"
+              6468733269                       #           "dhs2i"
+  6b                                           #   text(11)
+     756e70726f746563746564                    #     "unprotected"
+  a0                                           #   tag(0)
+</pre>
+
+</section>
+
+
+</section>
+
+</section>
```

Comment:
This tag might not be properly formatted.  @selfissued could you help me clean it up and properly format it?